src/components/Icons.tsx

```diff
@@ -106,6 +106,7 @@ import XAU from '../../sass/icons/flags/xau.svg';
 import Russia from '../../sass/icons/flags/russia.svg';
 import Arabic from '../../sass/icons/flags/arabic.svg';
 import Bangladesh from '../../sass/icons/flags/bangladesh.svg';
+import Cambodia from '../../sass/icons/flags/cambodia.svg';
```

Comment:
aren't we using quill-icons for flags?

Reply:
In SmartCharts we're still using svg icons 🙏